.gitignore

```diff
@@ -0,0 +1,5 @@
+# .gitignore
+node_modules/
+.hintrc
+.stylelintrc.json 
```

Comment:
- [ ] The `.hintrc` and `.stylelintrc.json` files should not be added to the .gitignore file kindly remove it